reana_workflow_controller/workflow_run_manager.py

```diff
@@ -467,3 +457,19 @@ def _create_job_controller_startup_cmd(self, user=None):
             return [full_cmd]
         else:
             return base_cmd.split()
+
+    def _get_volume_mounts(self):
```

Comment:
Would [reana_commons.k8s.volumes:get_shared_volume](https://github.com/reanahub/reana-commons/blob/2a5bb3d85d71c12a93541357f10bb7e46bdd6dbe/reana_commons/k8s/volumes.py#L64) be of help here? if not we could adapt it, note that there we are changing depending of the deployment type CEPH or local Minikube, something we miss here.